debian/kano-vnc-i18n-orig.install

```diff
@@ -0,0 +1 @@
+po/messages.pot /mnt/translations/translations/kano-vnc/
```

Comment:
My only concern here is we are populating a hardcoded mount point.
How are we supposed to explain what to do to anyone who wants to clone and `debuild` on their own.